styles.css

```diff
@@ -0,0 +1,176 @@
+body {
+  font-family: "Josefin Sans";
+  background-color: #fffafb;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
```

Comment:
I suggest you give space before writing the next selector. This will make your code more readable 😊 .